packages/dds/tree/src/feature-libraries/flex-tree/flexTreeTypes.ts

```diff
@@ -533,7 +533,7 @@ export type FlexTreeObjectNodeFieldsInner<TFields extends FlexObjectNodeFields>
 		// Setter method (when the field is of a kind that has a logical set operation).
 		readonly [key in keyof TFields as TFields[key]["kind"] extends AssignableFieldKinds
 			? `set${Capitalize<key & string>}`
-			: never]: (content: FlexibleFieldContent) => void;
+			: never]: (content: FlexibleNodeContent) => void;
```

Comment:
Object fields cannot be sequence fields, so this can be simplified from "field content" to the more restrictive "node content". Also, this codepath is not used in production and is going to be deleted very soon anyway.